frontend/app/routes/_public+/apply+/$id+/applicant-information.tsx

```diff
@@ -139,8 +141,14 @@ export default function ApplyFlowApplicationInformation() {
           errorMessage={errorMessages.maritalStatus}
         />
         <div className="flex flex-wrap items-center gap-3">
-          <Button variant="alternative">{t('applicant-information.back-btn')}</Button>
-          <Button variant="primary">{t('applicant-information.continue-btn')}</Button>
+          <ButtonLink id="back-button" to={`/apply`}>
```

Comment:
No need for JSX curly braces unless the url path is not good and requires `id`.

```suggestion
          <ButtonLink id="back-button" to="/apply">
```

Reply:
Should be `"/apply"` and not `"apply"`. [Link](https://remix.run/docs/en/main/components/link#link)